openfisca_core/populations.py

```diff
@@ -204,10 +232,28 @@ def get_rank(self, entity, criteria, condition = True):
         # Return -1 for the persons who don't respect the condition
         return np.where(condition, result, -1)
 
+    def if_(self, condition: np.ndarray[bool], formula_if_true: Callable, formula_if_false: Optional[Callable] = None) -> np.ndarray:
```

Comment:
This helper is using the split and combine approach for a ternary condition case.

This doesn't necessarily need to  be merged in this PR, but is an example of what kind of helpers could be built on subpopulations.
There is for sure discussions to have about the interface we want to offer formula writers